core/kazoo_apps/src/kapps_notify_publisher.erl

```diff
@@ -97,7 +97,7 @@ maybe_handle_error(NotifyType, Req, Error) ->
 %% @doc save pay load to db to retry later
 -spec handle_error(ne_binary(), api_terms(), any()) -> 'ok'.
 handle_error(NotifyType, Req, Error) ->
-    lager:error("attempt for publishing notifcation ~s was unsuccessful: ~p", [NotifyType, Error]),
+    lager:warning("attempt for publishing notifcation ~s was unsuccessful: ~p", [NotifyType, Error]),
```

Comment:
I don't expect it to work here, there is no smtp server.

But what triggered `service_added` notification? From log it appears it triggered after `sup kapps_maintenance migrate`.

Also even without a smtp server, teletype should respond with a message says timeout to connect to smtp or something like that. This is weird yes, though it could be a race condition with initializing kazoo, creating the very first account and migration.